### ISSUE TITLE
signalStat with bad reception

### ISSUE THREAD

**Issue by soeadm** [panoramicdata/Meraki.Api#117]
One of my appliances has bad callular reception, so it often fall back to 2G and not 4G as primary used. When the Appliance uses 2G  the API returns  "signalStat" as an empty Array. Which leads to a crash i guess since signalStat is not set as List.

Using:  
/organizations/{organizationId}/appliance/uplink/statuses

This is returned for appliance connected to 2G cellular
{
                "interface": "cellular",
                "status": "ready",
                "ip": "xx.xx.xxx.xx",
                "provider": "tdc",
                "publicIp": null,
                "model": "pls8-e",
                **"signalStat": [],**
                "connectionType": null,
                "apn": null,
                "gateway": null,
                "dns1": null,
                "dns2": null,
                **"signalType": "2G",**
                "iccid": null
            }
This is an example of a appliance connected to 4G cellular, here signalStat works fine.
{
                "interface": "cellular",
                "status": "ready",
                "ip": "xx.xx.xxx.xx",
                "provider": "tdc",
                "publicIp": null,
                "model": "pls8-e",
                "signalStat": {
                    "rsrp": "-81",
                    "rsrq": "-6"
                },
                "connectionType": null,
                "apn": null,
                "gateway": null,
                "dns1": null,
                "dns2": null,
                "signalType": "4G",
                "iccid": null
            }

**Comment by danielabbatt** [comment]
Yes the API should not be returning different types for the same attribute., Not a library problem.
